controllers/storagecluster/storageclient.go

```diff
@@ -30,11 +28,10 @@ func (s *storageClient) ensureCreated(r *StorageClusterReconciler, storagecluste
 	}
 
 	storageClient := &ocsclientv1a1.StorageClient{}
-	storageClient.Name = storagecluster.Name + "-client"
-	storageClient.Namespace = r.OperatorNamespace
+	storageClient.Name = storagecluster.Name
 	_, err := cutil.CreateOrUpdate(r.ctx, r.Client, storageClient, func() error {
-		if err := controllerutil.SetControllerReference(storagecluster, storageClient, r.Scheme); err != nil {
```

Comment:
`controllerutil` is the more common name used in many repos. Can we please align on this and not `cutils`?

Reply:
done.